src/swap/components/SwapSettingsSlippageLayout.tsx

```diff
@@ -0,0 +1,62 @@
+import { Children, cloneElement, isValidElement, useMemo } from 'react';
+import { findComponent } from '../../internal/utils/findComponent';
+import { cn } from '../../styles/theme';
+import type { SwapSettingsSlippageLayoutReact } from '../types';
+import { SwapSettingsSlippageDescription } from './SwapSettingsSlippageDescription';
+import { SwapSettingsSlippageInput } from './SwapSettingsSlippageInput';
+import { SwapSettingsSlippageTitle } from './SwapSettingsSlippageTitle';
+import { SwapSettingsSlippageToggle } from './SwapSettingsSlippageToggle';
+
+export function SwapSettingsSlippageLayout({
+  children,
+  className,
+  customSlippageEnabled,
+  onToggleCustomSlippage,
```

Comment:
Listeneres are not allowed at component, level, and **all listeners** have to be controled by the `onStatus` lifecycle hook.

Reply:
removed

Reply:
will start to think Life Cycle Status first